financialaid/permissions.py

```diff
@@ -23,3 +23,20 @@ def has_object_permission(self, request, view, obj):
             boolean
         """
         return has_object_permission(Permissions.CAN_EDIT_FINANCIAL_AID, request.user, obj.tier_program.program)
+
+
+class UserCanIndicateDocumentsSent(BasePermission):
```

Comment:
please rename to `RequestUserMatchesLoggedInUser` or something like that since we're 'generalizing' this functionality, and update docstrings